src/main/flight/mixer_init.c

```diff
@@ -50,7 +50,8 @@ PG_RESET_TEMPLATE(mixerConfig_t, mixerConfig,
     .mixerMode = DEFAULT_MIXER,
     .yaw_motors_reversed = false,
     .crashflip_motor_percent = 0,
-    .crashflip_expo = 35
+    .crashflip_expo = 35,
+    .linear_mixer = true
```

Comment:
A comma at the end of the line will make future pull requests easier to review.

Reply:
ok...will add a comma